env/env_test.go

```diff
@@ -87,6 +90,7 @@ func (s *MySuite) TestLoadDefaultEnvEvenIfDefaultEnvNotPresent(c *C) {
 
 	c.Assert(e, Equals, nil)
 	c.Assert(os.Getenv("gauge_reports_dir"), Equals, "reports")
+	c.Assert(os.Getenv("gauge_environment"), Equals, "default")
```

Comment:
why not using `common.DefaultEnvDir`?

Reply:
Same as before

Reply:
this test is confusing, it said `TestLoadDefaultEnvEvenIfDefaultEnvNotPresent` but indeed it loads the "default" env

Reply:
Done.